docs/content/latest/yugabyte-cloud/cloud-faq.md

```diff
@@ -143,9 +137,12 @@ For multi-region deployments, including [synchronous replication](../../explore/
 
 Upgrades are automatically handled by Yugabyte. There are two types of upgrades:
```

Comment:
If there are any differences between free and standard clusters, definitely call out how each one works. And if not, call out that this applies to both.

Reply:
There will be - db upgrades and selection in upcoming PR